src/firefly/js/api/ApiViewer.js

```diff
@@ -306,7 +307,7 @@ function buildChartPart(channel,file,dispatch) {
     /**
      * @summary Show Histogram
      * @param {HistogramOptions} histogramOptions
-     * @param {string} viewerId
+     * @param {string} viewerId - ignored in triview mode
```

Comment:
Are these viewerIds being ignored in triview because it creates its own ID for each viewer that is a part of it?

Reply:
yes, the tri-view manages the viewer id names. So the api has to send the predefined tri-view ids